hpedockerplugin/hpe_storage_api.py

```diff
@@ -450,7 +450,7 @@ def volumedriver_create(self, name, opts=None):
         valid_volume_create_opts = ['mount-volume', 'compression',
                                     'size', 'provisioning', 'flash-cache',
                                     'cloneOf', 'snapshotOf', 'expirationHours',
-                                    'retentionHours', 'promote']
+                                    'retentionHours', 'promote', 'qos-name']

```

Comment:
some validation you can add for presence of only -o flash-cache=true and -o qos-name=<vv_set>

Reply:
or only -o qos-name=<vv_set> .. As Imran pointed out cloneOf, snapshotOf should'nt present in the CLI along with -o qos-name=<vv_set>